call.go

```diff
@@ -236,6 +237,13 @@ func (cc *ClientConn) invoke(ctx context.Context, method string, args, reply int
 		if _, err := res.Body.Read(p.header[:1]); err != io.EOF {
 			return Errorf(codes.Internal, "grpc: malformed response with extra data after first message")
 		}
+	} else {
+		if len(p.header) == 0 {
+			// We received 0 bytes
+			if !res.ProtoAtLeast(2, 0) {
+				return Errorf(codes.Internal, fmt.Sprintf("grpc: HTTP/2 is required, but %v is used", res.Proto))
```

Comment:
you don't need fmt.Sprintf. Errorf already does it.

Also, move this up to line 203-ish, right after you get the response